app/views/users/profile.html.erb

```diff
@@ -233,12 +233,17 @@
       </div>
     </div>
   <% end %>
+  
+
+    <% if @profile_user && current_user && current_user.can_moderate? && @profile_user == current_user %>
+
+      <%= form_tag "/users/test_digest_email", method: :post do %>
+        <%= submit_tag "Test Digest Email", class: "btn btn-primary" %>
```

Comment:
```suggestion
        <%= submit_tag "Test Digest Email", class: "btn btn-secondary" %>
```

As secondary would look little lighter. What do you think?

Reply:
the btn-secondary bootstrap class is grayish, but the btn-info in a bit lighter than the primary. I have gone with that.